### DISCUSSION TITLE
How do to properly create a vde network?

### DISCUSSION THREAD

**Discussion by mike-cuche** [86Box/86Box#4100] [Q&A] [answered]
Hello,
I'm trying to create a VDE network (on Arch Linux) that connects to both the host and the internet (and if possibile also to the virt-manager bridge for accessing modern VMs) but with I had no luck, documentation is scarce.
Do someone have some insights for properly doing that?

**Comment by pshirshov** [comment, marked as answer]
On your Linux host:

```
BR_INET=br0
BOX_TAP=tap0
﻿ip tuntap add mode tap $BOX_TAP
﻿ip link set dev $BOX_TAP up
brctl addif $BR_INET $BOX_TAP
vde_switch --mode 666 --numports 8 --mgmt /tmp/vde.mgmt --mgmtmode 666 -s /tmp/vde.ctl -tap tap0
```

On your 86box host (including Macs):

```
LINUX_ROUTER=root@router
vde_switch --mode 666 --numports 8 --mgmt /tmp/vde.mgmt --mgmtmode 666 -s /tmp/vde.ctl
dpipe vde_plug /tmp/vde.ctl = ssh $LINUX_ROUTER vde_plug /tmp/vde.ctl
```

Set 86box to use `/tmp/vde.ctl`

It won't be fast (expect 100-200 KiB/s) but it will work.
